test_infra/stacks/databases_stack.py

```diff
@@ -493,7 +493,7 @@ def _setup_mysql(self) -> None:
                 "USERNAME": self.db_username,
                 "PASSWORD": self.db_password,
                 "JDBC_ENFORCE_SSL": "true",
-                "CUSTOM_JDBC_CERT": "s3://rds-downloads/rds-combined-ca-bundle.pem",
```

Comment:
Does this work? The documentation says that this needs to be an S3 location:
> CUSTOM_JDBC_CERT - An Amazon S3 location specifying the customer's root certificate. AWS Glue uses this root certificate to validate the customer’s certificate when connecting to the customer database. AWS Glue only handles X.509 certificates. The certificate provided must be DER-encoded and supplied in Base64 encoding PEM format.

Reply:
Thanks, updated